src/components/Accordion/AccordionItem/AccordionItem.js

```diff
@@ -0,0 +1,61 @@
+import React from 'react';
+import cx from 'classnames';
+import PropTypes from 'prop-types';
+import './AccordionItem.scss';
+
+const CN = 'accordion-item';
+
+const AccordionItem = ({
+  hide,
+  onClick,
+  title,
+  description,
+  height,
+  scroll
+}) => {
+  const heightStyle = {
+    height
+  };
+
+  return (
+    <div className={CN}>
+      <div
+        className={cx(`${CN}__header`, { [`${CN}__header--active`]: !hide })}
```

Comment:
`{ [`${CN}__header--active`]: !hide }` -- does this work without `[]` ?

Reply:
I tried and it does not work.